### PR TITLE
input variables for BDT

### PR DESCRIPTION
**Pull request by VincenzoCa**
- KEE and KMuMu variables stored now in the same output for data

- Add two functions, KEEcut and KMMcut

### CODE REVIEW

File: RDFAnalysis/RDF_MC_Kll_jobs.cpp [hunk 1]
```diff
@@ -52,16 +56,17 @@ int main(int argc, char **argv){
...
   std::vector<std::string> inputFileList;
-
+  
   if(inList != "-1"){
     std::string rootFileName;
     std::ifstream inFileLong;
     inFileLong.open(inList.c_str(), std::ios::in);
     while(!inFileLong.eof()){
       inFileLong >> rootFileName;
+      if( inFileLong.eof() ) break;
```
Comment (amartelli) [inline RDFAnalysis/RDF_MC_Kll_jobs.cpp:69]: didi you experience problem without this? 
in principle it's useless, but if this solves problems ok

**Comment by amartelli** [comment]
ok! few comments. to be taken into account in the next PR.
can merge

File: RDFAnalysis/RDF_MC_Kll_jobs.cpp (hunk 1 above)
Comment (VincenzoCa) [reply RDFAnalysis/RDF_MC_Kll_jobs.cpp:69]: Without `if( inFileLong.eof() ) break;` it reads the last line of the input file list twice

Comment (amartelli) [reply RDFAnalysis/RDF_MC_Kll_jobs.cpp:69]: ok thanks! surprising, but good if this solves

**Pull request merged**